app/client/components/pages/singleBulkDisbursementPage.jsx

```diff
@@ -98,19 +100,30 @@ class SingleBulkDisbursementPage extends React.Component {
 
     let status = bulkItem && bulkItem.state;
     let transferType = bulkItem && bulkItem.transfer_type;
-    let creator_user = bulkItem && bulkItem.creator_user;
+    let creatorUser = bulkItem && bulkItem.creator_user;
+    let approvalTimes = bulkItem && bulkItem.approval_times;
     let approvers = (bulkItem && bulkItem.approvers) || [];
     let label = bulkItem && bulkItem.label;
     let notes = bulkItem && bulkItem.notes;
     const approversList = approvers.map((approver, index, approversList) => {
       const spacer = index + 1 == approversList.length ? "" : ", ";
+      const approvalTimeString = approvalTimes[index]
+        ? " at " +
+          moment
+            .utc(approvalTimes[index])
+            .local()
+            .format("YYYY-MM-DD HH:mm:ss")
+        : "";
       return (
-        <a
-          style={{ cursor: "pointer" }}
-          onClick={() => this.navigateToUser(approver && approver.id)}
-        >
-          {approver && " " + approver.email + spacer}
-        </a>
+        <div>
+          <a
+            style={{ cursor: "pointer" }}
+            onClick={() => this.navigateToUser(approver && approver.id)}
+          >
+            {approver && " " + approver.email}
+          </a>
+          {approvalTimeString + spacer}
+        </div>
```

Comment:
Approver doesn't show for rejected transfers?

Reply:
This was the case before as well-- added the feature though!